### PR TITLE
prerelease changes

### PR DESCRIPTION
**Pull request by jani270**

### CODE REVIEW

File: src/main/java/io/github/moulberry/notenoughupdates/dungeons/GuiDungeonMapEditor.java [hunk 1]
```diff
@@ -284,7 +284,7 @@ public void drawScreen(int mouseX, int mouseY, float partialTicks) {
 							break;
 						case 8:
 							mapDesc = "Simple gray border";
-							mapCredit = "TomEngMaster";
+							mapCredit = "Madeline";
```
Comment (romangraef) [inline src/main/java/io/github/moulberry/notenoughupdates/dungeons/GuiDungeonMapEditor.java:287]: Shouldnt it be [Madeleaan](https://namemc.com/profile/Madeleaan.1)

Comment (jani270) [reply src/main/java/io/github/moulberry/notenoughupdates/dungeons/GuiDungeonMapEditor.java:287]: ive just copied her discord name

**Review by Lulonaut — DISMISSED** [review]

**Review by DeDiamondPro — DISMISSED** [review]

**Review by DeDiamondPro — APPROVED** [review]
Lil jani album drop when????

**Comment by jani270** [comment]
> Lil jani album drop when????

after janimod release

**Review by Lulonaut — APPROVED** [review]

**Review by NopoTheGamer — APPROVED** [review]
I can't wait to comment 2.2.md 🙂 on people's prs

**Pull request merged**